src/models/newsletters.ts

```diff
@@ -46,38 +46,38 @@ const baseNewsletterModel = {
 	brazeSubscribeAttributeNameAlternate: optional(t.array(t.string)),
 	brazeSubscribeEventNamePrefix: optional(t.string),
 	brazeNewsletterName: optional(t.string),
-	emailEmbed: EmailEmbedType,
+	emailEmbed: EmailEmbedCodec,
 	restricted: t.boolean,
 	paused: t.boolean,
 	emailConfirmation: t.boolean,
 	signupPage: optional(t.string),
 	exampleUrl: optional(t.string),
-	illustration: optional(NewsletterIllustrationType),
+	illustration: optional(NewsletterIllustrationCodec),
 	campaignName: optional(t.string),
 	campaignCode: optional(t.string),
 };
 
-export const CancelledEmailNewsletterType = t.type({
-	...baseNewsletterModel,
-	cancelled: t.literal(true),
-});
-export const EmailNewsletterType = t.type({
+export const BaseNewsletterCodec = t.type({ ...baseNewsletterModel });
+export type BaseNewsletter = t.TypeOf<typeof BaseNewsletterCodec>;
+
+export type NewsletterIllustration = t.TypeOf<
+	typeof NewsletterIllustrationCodec
+>;
+
+export const NewsletterResponseCodec = t.type({
```

Comment:
```suggestion
// See corresponding scala definition in the frontend project
// https://github.com/guardian/frontend/blob/c70a2d5d1a1374e0de0e9cf408116c7b76569bd0/common/app/services/newsletters/model/NewsletterResponse.scala
export const NewsletterResponseCodec = t.type({
```
Great idea to use the same name for the type as we have in other repos.
Just a suggestion - might be helpful for reference to link to the main consumer of the data.